src/web/components/ReaderRevenueLinks.tsx

```diff
@@ -132,74 +132,98 @@ export const ReaderRevenueLinks: React.FC<Props> = ({
     dataLinkNamePrefix,
     inHeader,
 }) => {
+    const [isDigitalSubscriber, setIsDigitalSubscriber] = useState<boolean>(
+        false,
+    );
+    const [hideSupportMessage, setShouldHideSupportMessage] = useState<boolean>(
+        false,
+    );
+
     const [isPayingMember, setIsPayingMember] = useState<boolean>(false);
+
     const [isRecentContributor, setIsRecentContributor] = useState<boolean>(
         false,
     );
 
     useEffect(() => {
         // Is paying member?
         setIsPayingMember(getCookie('gu_paying_member') === 'true');
+
+        // Should the support messages be shown
+        setShouldHideSupportMessage(
+            getCookie('gu_hide_support_messaging') === 'true',
+        );
+
         // Is recent contributor?
         setIsRecentContributor(decideIfRecentContributor());
+
+        // Is digital subscriber?
+        setIsDigitalSubscriber(getCookie('gu_digital_subscriber') === 'true');
     }, []);
 
-    if (!isPayingMember && !isRecentContributor) {
-        return (
-            <div className={cx(inHeader && paddingStyles)}>
-                <div
-                    className={cx({
-                        [hiddenUntilTablet]: inHeader,
-                    })}
+    /*
+        Changed to OR statement as it's likely that at least one will will be false.
+        Default response is to show the banners
+    */
+    if (
+        isDigitalSubscriber ||
+        isPayingMember ||
```

Comment:
you don't really need these as the `gu_hide_support_messaging` intentionally aggregates all the different products into a single flag (see https://github.com/guardian/members-data-api#dotcom) so the control over this moves into the Reader Revenue sphere (makes it easy for us to add/recognise new products etc)
```suggestion
```

Reply:
note though that `isRecentContributor` is still required though as this helps to recognise contributors who bought using 'guest checkout' (not signed-in, all other products require sign-in to purchase)